packages/react-badge/src/Badge.stories.tsx

```diff
@@ -1,60 +1,36 @@
 import * as React from 'react';
-import { Badge } from './index';
+import { Badge, BadgeProps } from './index';
+import descriptionMd from './BadgeDescription.md';
+import bestPracticesMd from './BadgeBestPractices.md';
+import { BadgeSize } from './components/Badge/Badge.types';
 
-export const BadgeAppearanceExample = () => (
-  <>
-    <h2>Circular (default)</h2>
-    <div
-      style={{
-        display: 'flex',
-        justifyContent: 'space-between',
-        alignItems: 'center',
-      }}
-    >
-      <Badge size="smallest" />
-      <Badge size="smaller" />
-      <Badge size="small" />
-      <Badge size="medium" />
-      <Badge size="large" />
-      <Badge size="larger" />
-      <Badge size="largest" />
-    </div>
-    <h2>Square</h2>
-    <div
-      style={{
-        display: 'flex',
-        justifyContent: 'space-between',
-        alignItems: 'center',
-      }}
-    >
-      <Badge size="smallest" shape="square" />
-      <Badge size="smaller" shape="square" />
-      <Badge size="small" shape="square" />
-      <Badge size="medium" shape="square" />
-      <Badge size="large" shape="square" />
-      <Badge size="larger" shape="square" />
-      <Badge size="largest" shape="square" />
-    </div>
-    <h2>Rounded</h2>
-    <div
-      style={{
-        display: 'flex',
-        justifyContent: 'space-between',
-        alignItems: 'center',
-      }}
-    >
-      <Badge size="smallest" shape="rounded" />
-      <Badge size="smaller" shape="rounded" />
-      <Badge size="small" shape="rounded" />
-      <Badge size="medium" shape="rounded" />
-      <Badge size="large" shape="rounded" />
-      <Badge size="larger" shape="rounded" />
-      <Badge size="largest" shape="rounded" />
-    </div>
-  </>
-);
+export const Template = (args: BadgeProps) => <Badge {...args} />;
+Template.args = {} as BadgeProps;
+
+export const {
+  smallest: Smallest,
+  smaller: Smaller,
+  small: Small,
+  medium: Medium,
+  large: Large,
+  larger: Larger,
+  largest: Largest,
+} = (['smallest', 'smaller', 'small', 'medium', 'large', 'larger', 'largest'] as BadgeSize[]).reduce((acc, curr) => {
+  acc[curr] = Template.bind({});
+  acc[curr].args = {
+    size: curr,
+  };
+  return acc;
+}, {} as Record<BadgeSize, typeof Template>);
 
 export default {
-  title: 'Components/Badge',
+  title: 'Components/Badge/Sizes',
   component: Badge,
+  parameters: {
+    docs: {
+      description: {
+        component: [descriptionMd, bestPracticesMd].join('\n'),
```

Comment:
should the component best practices really be a part of the description for how to use size ?